.github/workflows/publish-release.yml

```diff
@@ -113,6 +113,28 @@ jobs:
           cd installers/linux-rpm
           ./build-ballerina-linux-rpm-x64.sh -v ${{ steps.version-set.outputs.version }} -p ./../../ballerina/build/distributions
           echo "Created linux-rpm successfully"
+      - name: Generate Hashes
+        run: |
+          openssl dgst -sha256 -out ballerina-linux-installer-x64-${{ steps.version-set.outputs.version }}-deb.sha256 installers/linux-deb/target/ballerina-linux-installer-x64-*.deb | openssl enc -base64
+          openssl dgst -sha256 -out ballerina-linux-installer-x64-${{ steps.version-set.outputs.version }}-rpm.sha256 installers/linux-rpm/rpmbuild/RPMS/x86_64/ballerina-linux-installer-x64-*.rpm | openssl enc -base64
+      - name: Upload Linux deb Hashes
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_name: ballerina-linux-installer-x64-${{ steps.version-set.outputs.version }}-deb.sha256
```

Comment:
@NipunaMadhushan 
we can keep the same installer name and include .sha256 as the extension like `ballerina-linux-installer-x64-swan-lake-beta4.deb.sha256`